<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>731</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter Commande</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>411</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajout</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID_COMMANDE</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>PRIX</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>DATE_COM</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>QT_COMMANDES</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prix">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_DATE">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_Quantite">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_Ajouter">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>140</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Afficher Commande</string>
     </attribute>
     <widget class="QTableView" name="tableView_aff">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>411</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Supprimer Commande</string>
     </attribute>
     <widget class="QLabel" name="supp_text">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Saisie l'id du commande Ã  supprimer :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supp_Button">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="supprimer_id">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Modifier </string>
     </attribute>
     <widget class="QLineEdit" name="le_id_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ID_COMMANDE</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prix_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PRIX</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_DATE_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_Quantite_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>DATE_COM</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>QT_COMMANDES</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifierButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCommandes">
    <property name="title">
     <string>Commandes</string>
    </property>
   </widget>
   <addaction name="menuCommandes"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
